src/plugins/compact-view/config.ts

```diff
@@ -0,0 +1,62 @@
+import { ConfigItem } from "plugins";
+
+export const configList = [
+  {
+    type: "number",
+    key: "textFontSize",
+    default: 12,
+    min: 0,
+    max: 100,
+    step: 0.001,
```

Comment:
This (and later ones) should be step 1.

Reply:
fixed

I didn't change the size factor however since that's from 0 to 1